### ISSUE TITLE
Image Editor property keyboard accessibility

### ISSUE THREAD

**Issue by caesay** [clowd/Clowd#88]
- Spinner buttons should not be keyboard focusable (only the textbox)
- Color control is not keyboard focusable. When opened, should itself receive keyboard focus.
- Shift should not map to PAN so Shift-Tab works (we already have Space for PAN which is more standard anyway)
- There is some object in the bottom right quadrant of the canvas that should not be focusable
- Settings popout does not receive keyboard focus when opened
